mozilla/fx-private-relay
# for MPP-2223: add RelayNumber.enabled field

# New feature description
Adds a `PATCH /api/v1/relaynumber/{id}` endpoint to enable/disable the new `RelayNumber.enabled` field, which controls forwarding/blocking all calls and messages.

# Screenshot (if applicable)
<img width="711" alt="Screen Shot 2022-07-27 at 5 30 19 PM" src="https://user-images.githubusercontent.com/71928/181383635-2077e2c5-3865-4071-9cb7-159017eba652.png">

# How to test
1. Use the new endpoint at http://127.0.0.1:8000/api/v1/docs/ to set a relay number to `enabled: true`.
2. Send a text to the relay number
   * [x] It should be forwarded
3. Use the new endpoint at http://127.0.0.1:8000/api/v1/docs/ to set a relay number to `enabled: false`.
4. Send a text to the relay number
   * [x] It should NOT be forwarded

# Checklist
